infoblox_client/objects.py

```diff
@@ -1009,3 +1012,19 @@ class Tenant(InfobloxObject):
     _search_for_update_fields = ['id']
     _all_searchable_fields = _search_for_update_fields
     _shadow_fields = ['_ref']
+
+
+class CNAMERecord(InfobloxObject):
+    _infoblox_type = 'record:cname'
+    _fields = ['name', 'canonical', 'view', 'extattrs', 'comment',
+               'aws_rte53_record_info', 'cloud_info', 'creation_time',
+               'creator', 'ddns_principal', 'ddns_protected', 'disable',
+               'dns_canonical', 'dns_name', 'forbid_reclamation', 'reclaimable',
+               'ttl', 'use_ttl', 'zone']
+    _search_for_update_fields = ['name', 'view']
+    _updateable_search_fields = ['canonical', 'view', 'name', 'comment',
+                                 'creator', 'ddns_principal']
```

Comment:
_updateable_search_fields should just be a subset of _search_for_update_fields. 
Additionally, it is unlikely 'view' is updated. Therefore, _updateable_search_fields should only contain 'name'